Dockerfile

```diff
@@ -1,11 +1,5 @@
-FROM gradle:jdk11 as staticbuilder
-ADD . /build
-WORKDIR /build
-RUN gradle build --no-daemon
- 
-
-FROM openjdk:11-jdk as production
+FROM openjdk:11-jdk
 EXPOSE 8080
 VOLUME /tmp
-COPY --from=staticbuilder "/build/build/libs/haitaton-hanke-service-*-SNAPSHOT.jar" app.jar
-CMD [ "sh", "-c", "java $JAVA_OPTS -Djava.security.egd=file:/dev/./urandom -jar /app.jar" ]
+ADD "/build/libs/haitaton-hanke-service-*.jar" haitaton-hanke-service.jar
+CMD [ "sh", "-c", "java $JAVA_OPTS -Djava.security.egd=file:/dev/./urandom -jar /haitaton-hanke-service.jar" ]
```

Comment:
Apparently that /./ is no longer needed, /dev/urandom should do. A bit messy explanation here:
https://stackoverflow.com/questions/58991966/what-java-security-egd-option-is-for